### ISSUE TITLE
Not possible to close Mind Map pane in mobile version (Android)

### ISSUE THREAD

**Issue by ottantaventi** [lynchjames/obsidian-mind-map#51]
Using the plugin Mind Map v1.1.0 on my mobile Obsidian I’m able to correctly visualize the mind map, but there seems to be no way to close the map pane, when I no longer need the map.

**Comment by mkasprz** [comment]
That's true that this option is kind of missing from the pane menu, but it's possible to close it:
1. Focus Mind Map pane.
2. Open command palette by, for example, swiping from top of the application or tapping appropriate icon in left side menu.
3. Find "Close active pane" command and apply it.

You could also possibly use <kbd>Ctrl</kbd> + <kbd>w</kbd> hotkey if You use some external keyboard or [Hacker's Keyboard](https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard) (it's also available in F-Droid store) for example.
